src/schema/v2/partnerInquirerCollectorProfile.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  GraphQLBoolean,
+  GraphQLFieldConfig,
+  GraphQLFieldConfigMap,
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import date from "./fields/date"
+import Image, { normalizeImageData } from "./image"
+import { CollectorProfileFields } from "./me/collector_profile"
+import { myLocationType } from "./me/myLocation"
+
+export const CollectorProfileArtists = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "CollectorProfileArtists",
+  fields: {
+    name: { type: GraphQLString },
+  },
+})
+
+const InquirerCollectorProfileFields: GraphQLFieldConfigMap<
+  any,
+  ResolverContext
+> = {
+  ...CollectorProfileFields,
+  location: { type: myLocationType },
+  artsyUserSince: date,
+  ownerID: {
+    type: new GraphQLNonNull(GraphQLID),
+    description: "User ID of the collector profile's owner",
+    resolve: ({ owner: { id } }) => id,
+  },
+  icon: {
+    type: Image.type,
+    resolve: ({ icon }) => normalizeImageData(icon),
+  },
+  bio: {
+    type: GraphQLString,
+  },
+  profession: { type: GraphQLString },
+  otherRelevantPositions: {
+    type: GraphQLString,
+    description: "Collector's position with relevant institutions",
+    resolve: ({ other_relevant_positions }) => other_relevant_positions,
+  },
+  emailConfirmed: {
+    type: GraphQLBoolean,
+    resolve: ({ owner }) => owner.confirmed_at ?? false,
+  },
+  identityVerified: {
+    type: GraphQLBoolean,
+    resolve: ({ owner }) => owner.identity_verified,
+  },
+  isActiveInquirer: {
+    type: GraphQLBoolean,
+    resolve: ({ artwork_inquiry_requests_count }) =>
+      artwork_inquiry_requests_count >= 25,
+  },
+  isActiveBidder: {
+    type: GraphQLBoolean,
+    resolve: ({ previously_registered_for_auction }) =>
+      previously_registered_for_auction ?? false,
+  },
+  collectorProfileArtists: {
+    type: CollectorProfileArtists,
+    resolve: ({ collector_artists }) => collector_artists,
+  },
```

Comment:
I wonder if we can give this a more descriptive name. Are these actually artists the user collects or artists the user is interested in?

Reply:
for now, this is the list of artists the user is interested in, but later it will also include the list of artists collected by the user

Reply:
Ok, very difficult to find a good name for it. Maybe a good description works as well :)